jupyterlab_gcedetails/src/components/hardware_scaling_status.tsx

```diff
@@ -215,15 +262,21 @@ export class HardwareScalingStatus extends React.Component<Props, State> {
   }
 
   render() {
-    const { status } = this.state;
-    const progressValue = (status / 4) * 100;
+    const { status, error, instanceDetails } = this.state;
+    const progressValue = (status / 6) * 100;
     const { flexContainer, heading, paragraph } = STYLES;
     const { onDialogClose } = this.props;
-    return (
+    return status === 7 ? (
+      <ErrorPage
+        onDialogClose={onDialogClose}
+        error={error}
+        instanceDetails={instanceDetails}
+      />
+    ) : (
       <div className={flexContainer}>
         <h3 className={heading}>{Status[status]}</h3>
         <p className={paragraph}>{statusInfo[status]}</p>
-        {status === 4 || status === 5 ? (
+        {status === 6 ? (
```

Comment:
Can we refer to the Status by it's Enum value (`Status['Complete']` I believe)? Otherwise, it sort of defeats the purpose of using enums as named constants.